CMakeLists.txt

```diff
@@ -184,11 +119,6 @@ if(UNIX)
   set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Microsoft - Azure Storage")
   set(CPACK_DEBIAN_PACKAGE_DEPENDS "fuse")
   set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "blobfuse 1.2.4 - FUSE adapter for Azure Blob Storage")
-  set(VERSION "1.2.4")
-  set(CPACK_PACKAGE_VERSION_MAJOR 1)
```

Comment:
why are these removed. This will break the RPM generator. Also, the version should be 1.2.5

Reply:
I have moved this up towards the begining. Wanted to use the version defined here in the code directly instead of updating version string at multiple places for every release. I have bumped up version to 1.3.1, will reset to 1.2.5.

Reply:
Please try the RPM generator with latest code base in my master, shall work fine. If you see any breakage due to my changes do let me know.

Reply:
The code is not even compiling in RHEL, generaterpm.sh needs a compiled blobfuse.